frontend/src/App.js

```diff
@@ -28,9 +29,9 @@ const Routes = [
   <StatisticsPage path={routes.PATH_STATISTICS} />,
   <SettingsPage path={routes.PATH_SETTINGS} />,
   <AddTransactionPage path={routes.PATH_ADD_TRANSACTION} />,
-  <ResetPasswordPage path={routes.PATH_RESET_PASSWORD} />,
-  <ChooseTeamPage path={routes.PATH_CHOOSE_TEAM} />,
-  <CommentPage path="/comments/:comment" />,
+  // <ResetPasswordPage path={routes.PATH_RESET_PASSWORD} />,
+  // <ChooseTeamPage path={routes.PATH_CHOOSE_TEAM} />,
+  // <CommentPage path="/comments/:comment" />,
```

Comment:
Comments

Reply:
Temporary non functional code until the features are implemented in GraphQL.